service/web_svc.py

```diff
@@ -80,3 +87,116 @@ def __init__(self):
 
             return data
 
+    # async def find_full_source(self, images, source):
+    #     """"Expected input: images - list of full sources (including hostname); source - partial url (no hostname) """
+    #     for i in range(0, len(images)):
+    #         if source in images[i]:
+    #             return images[i]
+    #     return source
+
+    async def map_all_html(self, url_input):
+        a = newspaper.Article(url_input, keep_article_html=True)
+        a.download()
+        a.parse()
+        results = []
```

Comment:
these four can be moved to all one line
`results, plaintext, htmltext, images = [], [], [], []`